dependencies/cmake/mimalloc/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+include(FetchContent)
+FetchContent_Declare(
+    mimalloc
+    GIT_REPOSITORY https://github.com/microsoft/mimalloc.git
+    GIT_TAG v2.1.7 
+    GIT_SHALLOW TRUE
+)
+# if MI_OVERRIDE is ON then operator new and delete will be override not only malloc and free  
+# https://github.com/microsoft/mimalloc/issues/535
```

Comment:
If MI_OVERRIDE is ON then operator new and delete will be override not only malloc and free.
ref: https://github.com/microsoft/mimalloc/issues/535